File: DONGUK/프로그래머스_WEEK_14/이상한 문자 만들기.py
def solution(s):
    answer = ''
    
    cnt = 0
    for i in range(len(s)):
        if s[i] == ' ':
            answer += ' '
            cnt = 0
            continue
        
        if cnt % 2 == 1:
            c = s[i].lower()
            answer += c
            
        else:
            c = s[i].upper()
            answer += c
        
        cnt += 1
            
    return answer

File: JUNHEE/프로그래머스_week14/이상한 문자 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer = "";
    bool flag = false;
    int cnt = 0;
    for(int i = 0; i < s.size(); i++){
        if(cnt % 2 == 0){
            answer += toupper(s[i]);
        }
        else answer += tolower(s[i]);
        cnt++;
        if(s[i] == ' '){
            cnt = 0;
        }
    }
    return answer;
}
